core/design/src/main/kotlin/edu/stanford/spezi/core/design/component/CommonScaffold.kt

```diff
@@ -0,0 +1,53 @@
+package edu.stanford.spezi.core.design.component
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import edu.stanford.spezi.core.design.theme.Spacings
+import edu.stanford.spezi.core.design.theme.SpeziTheme
+import edu.stanford.spezi.core.design.theme.ThemePreviews
+
+@Composable
+fun CommonScaffold(
+    title: String,
+    content: @Composable (innerPadding: Modifier) -> Unit,
+) {
+    Scaffold(
+        topBar = {
+            AppTopAppBar(
+                title = {
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(end = Spacings.small),
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        Text(text = title)
+                    }
+                },
+            )
+        },
+        content = { innerPadding ->
+            Column(modifier = Modifier.padding(innerPadding)) {
+                content(Modifier.padding(innerPadding))
```

Comment:
Should't be need to pass this modifier with the inner padding to the content imo, `Modifier.fillMaxSize().padding(innerPadding)` should be enough. Also, since you are using a `Column` here, would be good to indicate to the calllers of the CommonScaffold by updating content type to `content: @Composable ColumnScope.() -> Unit,`. Also let's add a `modifier: Modifier = Modifier` to the common scaffold and pass it internally in the scaffold

Reply:
good idea, i have adapted it accordingly 👍